### PR TITLE
add builtin_gbdt_train python script

### PR DESCRIPTION
**Pull request by czzmmc**
## Description

Add builtin_gbdt_train python script to help people use python sdk. All my codes have been tested. But I **do not** add them into teaclave tests, because I dont know how to do that... So could u guys help me finish the final shot? BTW, builtin_gbdt_predict python script and codes about LR thing are on the way.

### CODE REVIEW

File: examples/python/builtin_gbdt_train.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        return bytes(result)
+
+
+def main():
+    example = BuiltinGbdtExample(USER_ID, USER_PASSWORD)
+    if len(sys.argv) > 1:
+        message = sys.argv[1]
+        rt = example.gbdt(message)
+    else:
+        rt = example.gbdt()
```
Comment (mssun) [inline examples/python/builtin_gbdt_train.py:134]: message is not needed for this example.
```
    if len(sys.argv) > 1:
        message = sys.argv[1]
        rt = example.gbdt(message)
    else:
        rt = example.gbdt()
```
=>
```
         rt = example.gbdt() 
```

File: examples/python/builtin_gbdt_train.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    def __init__(self, data_name, data_id):
+        self.data_name = data_name
+        self.data_id = data_id
+
+class BuiltinGbdtExample:
+    def __init__(self, user_id, user_password):
+        self.user_id = user_id
+        self.user_password = user_password
+
+    def gbdt(self, message="Hello, Teaclave!"):
```
Comment (mssun) [inline examples/python/builtin_gbdt_train.py:45]:
```
     def gbdt(self, message="Hello, Teaclave!"): 
```
=>
```
     def gbdt(self): 
```

File: sdk/python/teaclave.py [hunk 3]
```diff
@@ -181,7 +238,7 @@ def get_task_result(self, task_id):
             time.sleep(1)
             if response["content"]["status"] == 10:
                 break
-
+
```
Comment (mssun) [inline sdk/python/teaclave.py:241]: Remove these trailing whitespaces.

**Comment by mssun** [comment]
Please also use [yapf](https://github.com/google/yapf) to format the python code. Thanks.

**Comment by czzmmc** [comment]
@mssun Please recheck. It's done.

**Pull request merged**